src/components/Card.js

```diff
@@ -32,18 +48,24 @@ export default class Card {
       ".card__description-text"
     );
 
-    cardImageEl.src = this.link; //adds card image
-    cardImageEl.alt = this.name; //adds card alt text
-    cardNameTextEl.textContent = this.name; //Description
+    cardImageEl.src = this._link; //adds card image
+    cardImageEl.alt = this._name; //adds card alt text
+    cardNameTextEl.textContent = this._name; //Description
   }
 
   _handleLikeButton() {
+    if(this.isLiked)
+    {api.unlikeCard(this.id).then(json => this.liked = false)}
+    else
+    {api.likeCard(this.id).then(json => this.isLiked = true)};
     this._cardElement
       .querySelector(".card__like-button")
       .classList.toggle("card__like-button_active");
```

Comment:
You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.